README.md

```diff
@@ -52,17 +67,56 @@ Its use should be automated in Composer scripts.
 }
 ```
 
-or
+This provides `composer strauss`, which does the following:
+
+1. The `sh -c` command tests if `bin/strauss.phar` exists, and if not, downloads it from [releases](https://github.com/BrianHenryIE/strauss/releases).
```

Comment:
`test -f` tests does it exist. `sh -c` starts a new shell for the quoted commands.

Reply:
Yeah, the intent of the sh -c callout was to reference the line itself, not just the sh -c portion. Your point is good, though :)